src/website/app/demos/ThemeProvider/bestPractices.js

```diff
@@ -0,0 +1,49 @@
+export default [
+  {
+    type: 'do',
+    title: 'override theme variables at the top level',
+    example: `
+\`\`\`
+import { mineralTheme } from 'mineral-ui/utils';
+
+<ThemeProvider theme={{
+  ...mineralTheme,
+  color_primary: 'rebeccapurple'
+}}>
+  <App />
+</ThemeProvider>
+\`\`\`
+`,
+    description: `Try to keep theme overrides all in one place, as they will be easier to change later.`
```

Comment:
This is probably a good enough start, but we'll likely want to add more nuance here to account for different scenarios. For example, what if they want to both 1) change the default Button and 2) add a completely new style of Button.

Reply:
added another clarifying (hopefully) sentence.

Reply:
Yeah, that's a tough one. I still think it could be improved, but no need for it to be a blocker.